ricecooker/classes/nodes.py

```diff
@@ -114,28 +114,24 @@ class Channel(Node):
             thumbnail (str): file path or url of channel's thumbnail (optional)
     """
     thumbnail_preset = format_presets.CHANNEL_THUMBNAIL
-    def __init__(self, channel_id, domain, title, description=None, thumbnail=None):
+    def __init__(self, channel_id, source_domain, title, description="", thumbnail=None):
         # Map parameters to model variables
-        self.source_domain = domain
+        self.source_domain = source_domain
         self.source_id = channel_id
```

Comment:
Having these match would be good too (in general, for any `__init__` arg that's directly applied as a class attribute)